test/parallel/test-http2-session-timeout.js

```diff
@@ -5,6 +5,8 @@ if (!common.hasCrypto)
   common.skip('missing crypto');
 const assert = require('assert');
 const http2 = require('http2');
+const hrtime = process.hrtime.bigint;
+const NS_PER_MS = 1000000n;
```

Comment:
```suggestion
const NS_PER_MS = 1_000_000n;
```

Reply:
Applied your suggestion. Thank you for suggestion.